app/assets/stylesheets/css3/_transition.scss

```diff
@@ -58,9 +58,7 @@
 }
 
 @mixin transition-property ($properties...) {
-   -webkit-transition-property: transition-property-names($properties, 'webkit');
-      -moz-transition-property: transition-property-names($properties, 'moz');
-           transition-property: transition-property-names($properties, false);
+  @include prefixer(transform-property, $properties, webkit moz spec);
```

Comment:
I left it in since we are [supposedly supporting Firefox 3.6+](https://github.com/thoughtbot/bourbon#browser-support), and it's only supported the unprefixed version since version 16 (came out in August 2012).

It may be time to update that list though.